src/AdminApi/ScriptTag/Builders/GetScriptTagsRequest.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+namespace Yaspa\AdminApi\ScriptTag\Builders;
+
+use DateTime;
+use Yaspa\Constants\RequestBuilder;
+use Yaspa\Interfaces\PagingRequestBuilderInterface;
+use Yaspa\Traits\AuthorizedRequestBuilderTrait;
+
+/**
+ * Class GetScriptTagsRequest
+ *
+ * @package Yaspa\AdminApi\ScriptTags\Builders
+ * @see https://help.shopify.com/api/reference/scripttag#index
+ */
+class GetScriptTagsRequest implements PagingRequestBuilderInterface
+{
+    use AuthorizedRequestBuilderTrait;
+
+    const URI_TEMPLATE = 'https://%s.myshopify.com/admin/script_tags.json';
+
+    /** @var array|int[] $ids */
+    protected $ids;
+    /** @var int $limit */
+    protected $limit;
+    /** @var int $page */
+    protected $page;
+    /** @var int $sinceId */
+    protected $sinceId;
+    /** @var string $src */
+    protected $src;
+    /** @var string $event */
+    protected $event;
+    /** @var DateTime $createdAtMin */
+    protected $createdAtMin;
+    /** @var DateTime $createdAtMax */
+    protected $createdAtMax;
+    /** @var DateTime $updatedAtMin */
+    protected $updatedAtMin;
+    /** @var DateTime $updatedAtMax */
+    protected $updatedAtMax;
+    /** @var string $displayScope */
+    protected $displayScope;
```

Comment:
`displayScope` doesn't appear in list of fields in https://help.shopify.com/api/reference/scripttag#index